types/backend_url.pp

```diff
@@ -0,0 +1,5 @@
+type Sensu::Backend_URL = Variant[
```

Comment:
Check out this PR to stdlib as an example for writing spec tests for type aliases

https://github.com/puppetlabs/puppetlabs-stdlib/commit/6dad21e292436ac5b7d8113e81e572a0efc66782

Reply:
Added unit tests based on the stdlib example.